### DISCUSSION TITLE
Azure Rosja

### DISCUSSION THREAD

**Discussion by acidef** [AzureCommunityPL/group#327] [Pomoc]
Cześć,

czy jest możliwość postawienia SQL z alokacją w Rosji? 
Mamy potencjalnego klienta na usługę firma z Rosji i ich GDPR wymaga aby tam były przechowywane dane.
Wstępnie z tego co patrzyłem na portalu nie ma w regionie do wyboru rejonu, który by na to pozwalał.

Z gór dzięki.

**Comment by smereczynski** [comment]
Patrzyłeś w dokumentacji regionów? Z ciekawości pytam ;) Podpowiem, że Władek nie jest fanem amerykańskich korporacji ;)

**Reply by acidef** [reply]
nie patrzyłem jeszcze, mam zamiar to zrobić po spotkaniach :-) tak było szybciej ;p

**Reply by smereczynski** [reply]
OK, niemniej nie odpowiedziałem na Twoje pytanie. Jest możliwość. ARC-enabled Kubernetes + App Service z Custom Location na tymże K8S. Druga opcja to Azure Stack Hub. Jeśli pytasz o publiczny Azure i typowe App Service, to patrz odpowiedź wyżej ;)

**Reply by mgrabarz** [reply]
Ale wiecie, mam nadzieję, że WebApp nie służy do przechowywania danych? :D

**Reply by acidef** [reply]
hehe tak wiemy . Mój błąd w opisie. Pisane przed kawą ;p

**Reply by smereczynski** [reply]
> Ale wiecie, mam nadzieję, że WebApp nie służy do przechowywania danych? :D

A kto nam zabroni? :P

**Comment by mgrabarz** [comment]
Jeśli chodzi o tematykę danych osobowych, to niestety Rosja i Kazachstan robią mocno pod górę.

Nie ma tam regionu żadnego z dużych dostawców chmury, ale lokalne firmy "wyczuły biznes" i oferują przeróżne hostingi, Azure stacki itp by sprostać wymaganiom PII. W tym miejscu warto pogadać z "waszym" dostawcą IAM/CIAM, co rekomenduje swoim klientom w temacie Rosji. Tutaj porady od MS: https://docs.microsoft.com/en-us/compliance/regulatory/offering-russia-data-localization
